<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_method</class>
 <widget class="QWidget" name="add_method">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>262</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>456</width>
    <height>262</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加处置方式</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/input_tablet_32px.ico</normaloff>icons/input_tablet_32px.ico</iconset>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>序号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="line_deal_id">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>不合格品ID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="line_unpass_id">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>处理日期</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="date_deal">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2019</year>
        <month>6</month>
        <day>30</day>
       </datetime>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2017</year>
        <month>9</month>
        <day>14</day>
       </datetime>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2019</year>
        <month>6</month>
        <day>30</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>处理措施</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="txt_deal_method"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>处理数量Kg</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="line_deal_qty"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_save_method">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/confirm128px.ico</normaloff>icons/confirm128px.ico</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>icons/dealwith_128px.ico</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>date_deal</tabstop>
  <tabstop>line_deal_qty</tabstop>
  <tabstop>txt_deal_method</tabstop>
  <tabstop>btn_save_method</tabstop>
  <tabstop>line_deal_id</tabstop>
  <tabstop>line_unpass_id</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
